src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, Integer age) {
+        this.login = login;
+        this.password = password;
+        this.age = age;
+    }
+
```

Comment:
I have doubts that this is an unnecessary implementation.

Reply:
Thank you for comment. Constructor with arguments used in my data test sets. In my opinion, its use is reasonable.